src/Domain/Model/DynamoDbModel.php

```diff
@@ -183,6 +183,10 @@ public function save(array $options = [])
             $this->fireModelEvent('creating');
         }
 
+        if ($this->usesTimestamps()) {
```

Comment:
one additional note `$model->updateTimestamps()` is just updating the attributes and not really persisting them

    protected function updateTimestamps()
    {
        $time = $this->freshTimestamp();

        if (! $this->isDirty(static::UPDATED_AT)) {
            $this->setUpdatedAt($time);
        }

        if (! $this->exists && ! $this->isDirty(static::CREATED_AT)) {
            $this->setCreatedAt($time);
        }
    }